content/src/main/content/jcr_root/etc/acs-commons/httpcache/root/.content.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same as above..

/etc is falling out of favor, and /var is the spot for transient, system-managed content.